transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/SelectDestinationItemClickListener.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * The MIT License (MIT)
+ * Copyright (c) 2019 Hyperwallet Systems Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.hyperwallet.android.ui.transfer.view;
+
+import com.hyperwallet.android.model.transfermethod.HyperwalletTransferMethod;
+
+public interface SelectDestinationItemClickListener {
```

Comment:
Please move this interface inside the fragment since this is very specific to only selection of transfer destination, and we won't be sharing this in the future

Reply:
Is not possible in this case.